apps/change-log.md

```diff
@@ -14,6 +14,14 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2021-11-22
+
+#### <i class="fa fa-minus"></i> Deprecate
+
+* The `pageview`, `product`, `category`, `search`, and `cart` event in the [Events](https://developer.epages.com/apps/events.html) section are deprecated and will be removed on 22nd February 2022. Please use the respective alternatives `page:view`, `product:view`, `category:view`, `searchResults:view`, and `cart:view` instead.
```

Comment:
Two questions for this part:

1: Am I correct that the new events can already be used from now on, unrelated to the release version?

2: Is 22nd February a reasonable removal date? We should give them at least some information about how much time they have to adapt to the change. So we should make sure that we do not remove them earlier. Later would be fine.

Reply:
1. The new events are released with the 2.28.0 version which is in rollout at the moment.
2. That is about 2.5 months from now. How much time do we give partners when we deprecated APIs?
IMO 3 months, a quarter, is a good time frame. We can keep it longer if requested.

Reply:
1. I see. Well in that case it's always difficult to tell partners when this will be rolled out for which provider as we do not use this version in the change log / the docs. I'd suggest to mention the next release (7.66) here. Providers or merchants with that version should then definitely also have the 2.28.0. Please correct me if that assumption is wrong. I'll then merge the PR after my vacation (the whole next week) which would match the rollout start of the 7.66 version.

2: It's 2-3 months depending on the impact of the change. So yes, I agree. 3 months is a good time frame for this change.

Reply:
1. I think that the epages-ui versions are bound to certain ePages Now versions, so it should be safe. With the current release it will be included.